extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -290,4 +299,18 @@ export class DbPanel extends DisposableObject {
     // You can only select one item at a time, so selection[0] gives the selection
     return this.treeView.selection[0]?.dbItem;
   }
+
+  private async openOnGitHub(treeViewItem: DbTreeViewItem): Promise<void> {
+    if (treeViewItem.dbItem === undefined) {
+      throw new Error("Unable to open on GitHub. Please select a valid item.");
+    }
+    const githubUrl = getGitHubUrl(treeViewItem.dbItem);
+    if (!githubUrl) {
+      throw new Error(
+        "Unable to open on GitHub. Please select a remote repository or owner.",
+      );
+    }
+
+    await commands.executeCommand("vscode.open", Uri.parse(githubUrl));
```

Comment:
Instead of using Uri from vscode we could use URL (new URL('string')).

Reply:
I like this! But annoyingly it looks like `vscode.open` expects a Uri or string, so can't use URL 😿 

```
Invalid argument 'uriOrString' when running 'vscode.open', received: https://github.com/codeql (codeQLDatabasesExperimental.openOnGitHubContextMenu)
```